api/tests/src/db/models/factories.py

```diff
@@ -1833,3 +1835,27 @@ class Meta:
 
     opportunity = factory.SubFactory(OpportunityFactory)
     opportunity_id = factory.LazyAttribute(lambda s: s.opportunity.opportunity_id)
+
+
+class UserFactory(BaseFactory):
+    class Meta:
+        model = user_models.User
+
+    user_id = Generators.UuidObj
+
+
+class LinkExternalUserFactory(BaseFactory):
+    class Meta:
+        model = user_models.LinkExternalUser
+
+    link_external_user_id = Generators.UuidObj
+    external_user_id = "test111"
```

Comment:
You'll want to make this be a unique-ish value.

Since login.gov will be the only one and they use UUIDs, we should be able to reuse that here.

Reply:
using the `Generators.UuidObj` now